### PR TITLE
Add ability to show cc and bcc recipients if present, and not error on missing To recipient

### PR DESCRIPTION
**Pull request by compwron**
## Overview

Add ability to show bcc recipients if present, and not error on missing To recipient

## Notes

This codebase uses https://www.npmjs.com/package/mailparser and the docs there imply to me that this is possible. 

Command to run the tests on my local, thanks to denis90: `NODE_OPTIONS="$NODE_OPTIONS --experimental-vm-modules" npm test`

Refs: LM-zxy

### CODE REVIEW

File: test/app.test.js [hunk 1]
```diff
@@ -43,6 +43,19 @@ describe("App Tests", () => {
               html_part: null,
             },
           },
+          ,
+          {
+            Timestamp: Date.now(),
```
Comment (veelenga) [inline test/app.test.js:48]: seems to be a second comma in a row

File: views/index.pug [hunk 2]
```diff
@@ -61,7 +61,7 @@ html
                 div.logo
                   if logo
                     img(src=`data:image/png;base64,${logo.toString('base64')}`, height="24px", alt="Logo")
-            td= message.to
+            td= `${message.to} BCC: ${message.bcc}`
```
Comment (veelenga) [inline views/index.pug:64]: I think this needs to be improved, i.e if there is no BCC, it should not show up here. 
Also would be nice to have CC column as an addition

Comment (denis90) [reply views/index.pug:64]: Can we do bcc and cc as separate rows?

Comment (compwron) [reply views/index.pug]: Done as separate rows!

File: app.js [hunk 3]
```diff
@@ -106,7 +106,9 @@ async function createEmail(message) {
     return {
       timestamp: message.Timestamp,
       subject: parsed.subject,
-      to: parsed.to.text,
+      to: parsed.to?.text,
```
Comment (compwron) [inline app.js:109]: This should null values to not error the server, which is what brought me here in the first place :)

File: test/app.test.js (hunk 1 above)
Comment (compwron) [reply test/app.test.js]: fixed

**Comment by compwron** [comment]
@denis90 or @veelenga can you approve my PR GHA CI run and review/stamp? (Sorry to hurry you, I am just enthusiastic...) If you expect that it will take a few days or a week to get to this, I will just release a temporary npm fork build for my team :)

File: app.js (hunk 3 above)
Comment (veelenga) [reply app.js:128]: How can the email not have a to field?

**Review by denis90 — APPROVED** [review]
LGTM

**Pull request merged**